pkg/util/exec/exec.go

```diff
@@ -271,17 +252,12 @@ func runCommandWithOutput(actionName string, cmd *exec.Cmd, combinedOutput bool)
 	out := strings.TrimSpace(string(output))
 
 	if err != nil {
-		return out, createCommandError(err, actionName)
+		return out, err
 	}
 
 	return out, nil
 }
 
-func logCommand(debug bool, command string, arg ...string) {
-	msg := fmt.Sprintf("Running command: %s %s", command, strings.Join(arg, " "))
-	if debug {
-		logger.Debug(msg)
-	} else {
-		logger.Info(msg)
-	}
+func logCommand(command string, arg ...string) {
+	logger.Debugf("Running command: %s %s", command, strings.Join(arg, " "))
```

Comment:
Where do we log according to the log level defined in the operator log?

Reply:
The statement `logger.Debugf()` always logs at debug level. If you want to see this log message in the log, you have to enable `ROOK_LOG_LEVEL=debug` in the operator so that the debug level logging will be visible in the logs.

Reply:
Duh! 🤦‍♂ I guess it was still early when I did the review :-)